### ISSUE TITLE
Ng zorro select is not accessible via nvda reader

### ISSUE THREAD

**Issue by majdi91** [NG-ZORRO/ng-zorro-antd#7635]
## What problem does this feature solve?
Component accessibility: actually, nvda reader cannot read the status of the select component nether the options.

## What does the proposed API look like?
Should add aria-expanded for the select and aria-label for the options
